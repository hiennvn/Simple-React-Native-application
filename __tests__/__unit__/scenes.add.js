import 'react-native'
import React from 'react'
import configureStore from 'redux-mock-store'
import { shallow } from 'enzyme'
import AddScreen from '@scenes/Add'

const state = {}
const mockStore = configureStore()

describe('AddScreen', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<AddScreen store={mockStore(state)} />)
    expect(wrapper).toMatchSnapshot()
  })
})
