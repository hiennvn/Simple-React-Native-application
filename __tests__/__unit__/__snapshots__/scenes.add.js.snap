// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddScreen renders correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Add
    store={
        Object {
            "clearActions": [Function],
            "dispatch": [Function],
            "getActions": [Function],
            "getState": [Function],
            "replaceReducer": [Function],
            "subscribe": [Function],
          }
    }
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": null,
    "nodeType": "class",
    "props": Object {
      "children": <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        testID="add"
>
        [missing "en.add.welcome" translation]
</Text>,
      "style": Object {
        "backgroundColor": "#c764cc",
        "flex": 1,
      },
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": null,
      "nodeType": "class",
      "props": Object {
        "accessible": true,
        "allowFontScaling": true,
        "children": "[missing \\"en.add.welcome\\" translation]",
        "disabled": false,
        "ellipsizeMode": "tail",
        "testID": "add",
      },
      "ref": null,
      "rendered": "[missing \\"en.add.welcome\\" translation]",
      "type": [Function],
    },
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": null,
      "nodeType": "class",
      "props": Object {
        "children": <Text
          accessible={true}
          allowFontScaling={true}
          disabled={false}
          ellipsizeMode="tail"
          testID="add"
>
          [missing "en.add.welcome" translation]
</Text>,
        "style": Object {
          "backgroundColor": "#c764cc",
          "flex": 1,
        },
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": null,
        "nodeType": "class",
        "props": Object {
          "accessible": true,
          "allowFontScaling": true,
          "children": "[missing \\"en.add.welcome\\" translation]",
          "disabled": false,
          "ellipsizeMode": "tail",
          "testID": "add",
        },
        "ref": null,
        "rendered": "[missing \\"en.add.welcome\\" translation]",
        "type": [Function],
      },
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
